lib/Steps/PromptForParameters.ts

```diff
@@ -149,3 +158,39 @@ export class PromptForParameters extends BaseStep {
     return true;
   }
 }
+
+type MaybeOrgAuthToken = {
+  iat?: number;
+  url?: string;
+  org?: string;
+  region_url?: string;
+};
+
+/**
+ * Trying to parse and decode an org auth token. Based on:
+ * - https://github.com/getsentry/rfcs/blob/main/text/0091-ci-upload-tokens.md#parsing-tokens
+ * - https://github.com/getsentry/rfcs/blob/main/text/0091-ci-upload-tokens.md#token-facts
+ */
+function isValidOrgToken(input: string): boolean {
+  if (!input.startsWith('sntrys_')) {
+    return false;
+  }
+
+  const tokenParts = input.split('_');
+  if (tokenParts.length < 3) {
+    return false;
+  }
+
+  try {
+    const payload = tokenParts[1];
+    const decodedPayload = Buffer.from(payload, 'base64').toString();
+    const jsonPayload = JSON.parse(decodedPayload) as MaybeOrgAuthToken;
+    if (!jsonPayload.iat || !jsonPayload.url || !jsonPayload.org) {
```

Comment:
will `iat` and `url` always be there? Could they be null?

Reply:
According to https://github.com/getsentry/rfcs/blob/main/text/0091-ci-upload-tokens.md#token-facts they should always be there

Reply:
yes, they should, spec wise. we seem to have an issue where `url` is sometimes empty when running this locally, but this is something that we have to fix backend-side (and should not affect sentry.io)